fastplotlib/widgets/histogram_lut.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+
+from pygfx import Group
+
+from ..graphics import LineGraphic, ImageGraphic
+from ..graphics._base import Graphic
+from ..graphics.selectors import LinearRegionSelector
+
+
+# TODO: This is a widget, we can think about a BaseWidget class later if necessary
+class HistogramLUT(Graphic):
+    def __init__(
+            self,
+            data: np.ndarray,
+            image_graphic: ImageGraphic,
+            nbins: int = 100,
+            **kwargs
+    ):
+        super().__init__(**kwargs)
+
+        self.nbins = nbins
+        self._image_graphic = image_graphic
+
+        hist, edges, hist_scaled, edges_flanked = self._calculate_histogram(data)
+
+        line_data = np.column_stack([hist_scaled, edges_flanked])
+
+        self.line = LineGraphic(line_data)
+
+        bounds = (edges[0], edges[-1])
+        limits = (edges_flanked[0], edges_flanked[-1])
+        size = 120  # since it's scaled to 100
+        origin = (hist_scaled.max() / 2, 0)
+
+        self.linear_region = LinearRegionSelector(
+            bounds=bounds,
+            limits=limits,
+            size=size,
+            origin=origin,
+            axis="y",
+            edge_thickness=8
+        )
+
+        widget_wo = Group()
+        widget_wo.add(self.line.world_object, self.linear_region.world_object)
+
+        self._set_world_object(widget_wo)
+
+        self.world_object.local.scale_x *= -1
```

Comment:
is this for orientation of the histogram widget? I would maybe add comment

Reply:
So that the bins are left facing